docs/source/private-data/private-data.md

```diff
@@ -140,6 +143,184 @@ documentation on [transaction flow](../txflow.html).
    their copy of the private state database and private writeset storage. The
    private data is then deleted from the `transient data store`.
 
+## Sharing private data
+
+While you can create sets of multilateral private data collections to share data
+among collection members, this approach may result in a large number of collections
+that need to be defined. It also does not address the need to share or transfer
+private data beyond the collection members at the transaction or key level.
+For these reasons, consider using a private data collection per organization,
+and then sharing private data with other organization-specific collections as the
+need arises. Starting in Fabric v2.0, implicit organization-specific collections are
```

Comment:
Are implicit organization-specific collections described somewhere?  I might have missed it in the existing docs, but I did not see anything describing how to use/access them.

Reply:
I'll be using the same Jira issue to create reference doc for each of the v2.0 enhancements.